docs/node/network-migration.mdx

```diff
@@ -0,0 +1,92 @@
+
+```bash
+kwil-admin snapshot create --snapdir /path/to/snapdir
+
+# Running the above command will generate the snapshot file and the genesis.json file in the snapshot directory.
+ls /path/to/snapdir
+genesis.json            kwildb-snapshot.sql.gz
+```
+
+#### Verify App Hash
+
+App hash set in the genesis file is generated based on the hash of the uncompressed snapshot. You can verify the app hash in the generated genesis file as follows
+
+``` bash
+# Uncompress the snapshot file.
+gunzip /path/to/snapdir/kwildb-snapshot.sql.gz
+
+# Generate app hash using sha256sum.
+shasum -a 256 /path/to/snapdir/kwildb-snapshot.sql
+4c2ecfc145c37a45a1fd21cd38ee84db80d852996fc30438a007ba624387c683  /path/to/snapdir/kwildb-snapshot.sql
+
+# app_hash in genesis file is a hexadecimal representation of the sha256sum hash encoded in base64.
+# hex-> binary -> base64
+echo -n "4c2ecfc145c37a45a1fd21cd38ee84db80d852996fc30438a007ba624387c683" | xxd -r -p | base64
+
+TC7PwUXDekWh/SHNOO6E24DYUplvwwQ4oAe6YkOHxoM=
+
+# compare the app hash with the genesis file
+cat /path/to/snapdir/genesis.json | jq .app_hash
+
+"TC7PwUXDekWh/SHNOO6E24DYUplvwwQ4oAe6YkOHxoM="
+```
+
+To start the new network with the previous data, the genesis file must have the `app_hash` field set. If the `app_hash` field is not set, the network will start with an empty state.
+
+### Distribute Genesis State
+
+Distribute the `snapshot` and the `genesis.json` files required to start the new network. 
+
+Ensure that the `genesis.json` file has the `app_hash` field set to the hash of the uncompressed snapshot file. Refer to the [Verify App Hash](#verify-app-hash) section to correctly generate this hash. The genesis file is already set to reflect the genesis validators set to the previous network's validators set around the time of the snapshot.  If changes are needed for the genesis validators set or other configurations, update the genesis file manually as required before distributing it.
+
+### Run Genesis Nodes
+
+#### Install Kwild Binaries
+
+Firstly, install the new version of the `kwild` binaries. Refer to the [Installation](/docs//ref/installation) documentation for more details on getting the required versions of the `kwild` binaries.
+
+#### Download Genesis State
+
+Download the `snapshot` and the `genesis.json` files required for this new network. 
+
+#### Configure Node to Use Snapshot
+
+Refer to the [Node Configuration](/docs//ref/configuration) documentation to configure the `config.toml` file.
+
+To initialize the node with the snapshot data, the `snapshot_file` path in the `config.toml` file must be set to the location of the downloaded snapshot file.  The node will fail to start, if the `snapshot_file` path is not provided in the `config.toml`.
```

Comment:
your Node Config path seems to have an extra forward slash in there. Also, could you just show how to do this with flags instead?